multitenancy/src/main/java/org/axonframework/extensions/multitenancy/configuration/MultiTenantEventProcessingModule.java

```diff
@@ -48,9 +48,17 @@
 public class MultiTenantEventProcessingModule extends EventProcessingModule {
 
     private final TenantProvider tenantProvider;
+    private final MultiTenantStreamableMessageSourceConfiguration multiTenantStreamableMessageSourceConfiguration;
 
     public MultiTenantEventProcessingModule(TenantProvider tenantProvider) {
         this.tenantProvider = tenantProvider;
+        multiTenantStreamableMessageSourceConfiguration = ((defaultSource, processorName, tenantDescriptor, configuration) -> defaultSource);
+    }
+
+    public MultiTenantEventProcessingModule(TenantProvider tenantProvider,
```

Comment:
While we're here making changes any how, it would be great to add JavaDoc to both constructors.

Reply:
As well as providing some class-level JavaDoc to explain the intent of this module.